src/resources/i18n/task_fr.propeprties

```diff
@@ -0,0 +1,12 @@
+task_text = Texte
```

Comment:
The name of this file has a typo.
Change `src/resources/i18n/task_fr.propeprties` to `src/resources/i18n/task_fr.properties`